admin/add-ftp-page.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require('../template/top.php');
+head('Add FTP user', true);
```

Comment:
Probably need to have a discussion about what to do with the head() method for admin pages, typically I don't include it, but maybe we need something like `admin_head(..)` to load the admin styles.

Reply:
taken out, we can talk more about this later and in a future pr